openff/bespokefit/cli/executor/launch.py

```diff
@@ -0,0 +1,114 @@
+import time
+from typing import Optional
+
+import click
+import rich
+from click_option_group import optgroup
+from rich import pretty
+
+from openff.bespokefit.cli.utilities import create_command, print_header
+
+
+# The run command inherits these options so be sure to take that into account when
+# making changes here.
+def launch_options(
+    directory: str = "bespoke-executor",
+    n_fragmenter_workers: Optional[int] = None,
```

Comment:
Maybe we could have these default to 1, this would shorten the command a little when using `executor run` as well where we probably only ever want 1 fragmentation engine.